### PR TITLE
Adding new options to command-line interface

### PR DESCRIPTION
**Pull request by RyDroid**
https://github.com/phpmd/phpmd/pull/509

### CODE REVIEW

File: src/site/rst/documentation/index.rst [hunk 1]
```diff
@@ -40,10 +40,10 @@ Command line options
 
 - The command line interface also accepts the following optional arguments:
 
-  - ``--minimumpriority`` - The rule priority threshold; rules with lower
+  - ``--min-priority`` ``--minimum-priority`` ``--minimumpriority`` - The rule priority threshold; rules with lower
```
Comment (ravage84) [inline src/site/rst/documentation/index.rst]: I would prefer to have the original ones as first mentioned.

File: src/site/rst/documentation/index.rst [hunk 2]
```diff
@@ -40,10 +40,10 @@ Command line options
...
     priority than they will not be used.
 
-  - ``--reportfile`` - Sends the report output to the specified file,
+  - ``--report-file`` ``--reportfile`` - Sends the report output to the specified file,
```
Comment (ravage84) [inline src/site/rst/documentation/index.rst]: I would prefer to have the original one as first mentioned.

**Comment by RyDroid** [comment]
I reordered the options in the documentation.

**Comment by ravage84** [comment]
Thanks! 🍹

**Comment by RyDroid** [comment]
Thanks for your work too.

**Pull request merged**